presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayPositionFunction.java

```diff
@@ -44,23 +46,7 @@ public static long arrayPosition(
             @SqlType("array(T)") Block array,
             @SqlType("T") boolean element)
     {
-        int size = array.getPositionCount();
-        for (int i = 0; i < size; i++) {
-            if (!array.isNull(i)) {
-                boolean arrayValue = type.getBoolean(array, i);
-                try {
-                    Boolean result = (Boolean) equalMethodHandle.invokeExact(arrayValue, element);
-                    checkNotIndeterminate(result);
-                    if (result) {
-                        return i + 1; // result is 1-based (instead of 0)
-                    }
-                }
-                catch (Throwable t) {
-                    throw internalError(t);
-                }
-            }
-        }
-        return 0;
+        return ArrayPositionWithIndexFunction.arrayPositionWithIndex(type, equalMethodHandle, array, element, 1);
```

Comment:
Our coding convention is to use static import directly. So you should use
```
import com.facebook.presto.operator.scalar.ArrayPositionWIthIndexFunction.arrayPositionWithIndex;
```
However, I think it makes more sense to just merge the two classes.